test/micro/org/openjdk/bench/java/lang/StringIndexOfChar.java

```diff
@@ -41,18 +43,37 @@
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
 @State(Scope.Thread)
 public class StringIndexOfChar {
-    private static final int loops = 100000;
-    private static final Random rng = new Random(1999);
-    private static final int pathCnt = 1000;
-    private static final String [] latn1_short        = new String[pathCnt];
-    private static final String [] latn1_sse4         = new String[pathCnt];
-    private static final String [] latn1_avx2         = new String[pathCnt];
-    private static final String [] latn1_mixedLength  = new String[pathCnt];
-    private static final String [] utf16_short        = new String[pathCnt];
-    private static final String [] utf16_sse4         = new String[pathCnt];
-    private static final String [] utf16_avx2         = new String[pathCnt];
-    private static final String [] utf16_mixedLength  = new String[pathCnt];
-    static {
+    @Param("100000")
+    private int loops;
+
+    @Param("1000")
+    private int pathCnt;
+
+    @Param("1999")
+    private int rngSeed;
+
+    private Random rng;
+    private String [] latn1_short;
+    private String [] latn1_sse4;
+    private String [] latn1_avx2;
+    private String [] latn1_mixedLength;
+    private String [] utf16_short;
+    private String [] utf16_sse4;
+    private String [] utf16_avx2;
+    private String [] utf16_mixedLength;
```

Comment:
I'd be a little bit happier with `String[]` rather than `String []`

Reply:
Thanks Claes, I have made the modifications as advised